ui/app/routes/vault/cluster/secrets/backend/secret-edit.js

```diff
@@ -337,7 +337,7 @@ export default Route.extend(UnloadModelRoute, {
       // and explicity ignore it here
       if (
         (mode !== 'show' && changedKeys.length && changedKeys[0] !== 'backend') ||
-        (mode !== 'show' && version && Object.keys(version.changedAttributes()).length)
+        (mode !== 'show' && version && version.hasDirtyAttributes)
```

Comment:
This is interesting. `changedAttributes` doesn't seem to reset after saving, yet `hasDirtyAttributes` is returning `false`. In any case this is a more straightforward method to determine if the model is dirty.